tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = copying, test, code_check
+envlist = copying, test, code_check, control_node_serial
```

Comment:
It is related ?

Reply:
Yes, I think we should test the coverage of the C code as well